playbooks/roles/lets-encrypt/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+- include: install.yml
+
+- name: Prepare response file directory
+  file:
+    path: "{{ le_base }}/conf"
+    state: directory
+    recurse: yes
+
+- name: Put reponse file in place
+  template:
+    src: response.yaml.j2
+    dest: "{{ le_base }}/conf/responses"
+
+- name: Perform initial configuration of acmetool
+  command: acmetool quickstart
+  args:
+    creates: "{{ le_base }}/conf/target"
+
+# acmetool should now run in redirector mode listening on 80/tcp for HTTP
+# requests (including ACME chanllenges). All HTTP requests are simply
+# redirected to their HTTPS counterparts.
+
+- include: firewall.yml
+
+# Now we try to request a certificate. If it fails. We print a warning and
+# fall back to self-signed certificates.
+- block:
+    - name: Request initial certificate from Let's Encrypt
+      command: "acmetool want {{ streisand_domain }}"
+      args:
+        creates: "{{ le_certificate }}"
+
+    - set_fact:
+        le_ok: True
+
+  rescue:
+    - set_fact:
+        le_ok: False
```

Comment:
@cpu How did you skip the LE setup? The `le_ok` variable should always be set, as you could see here...